spec/features/providers/courses_as_an_accredited_body_spec.rb

```diff
@@ -35,10 +35,15 @@
       "#{accrediting_body1.provider_code}/training_providers?recruitment_cycle_year=#{accrediting_body1.recruitment_cycle.year}",
       resource_list_to_jsonapi([training_provider2, accrediting_body2]),
     )
+    stub_api_v2_request(
+      "/recruitment_cycles/#{accrediting_body1.recruitment_cycle.year}/providers/#{training_provider2.provider_code}" \
+      "/courses?filter[accrediting_provider_code]=#{accrediting_body1.provider_code}",
+      resource_list_to_jsonapi([course1]),
+    )
```

Comment:
Are we avoiding using `stub_api_v2_resource`?

Reply:
I don't think it supports `filter` qs params does it? We could add that I suppose...

Reply:
I think it'd be nice to have, assuming people are still using these stubs, but not crucial for this change.